awx/main/scheduler/task_manager.py

```diff
@@ -45,17 +46,46 @@
 class TaskManager():
 
     def __init__(self):
+        '''
+        Do NOT put database queries or other potentially expensive operations
```

Comment:
It's times like these that I employ one of my favorite comments:

https://github.com/ansible/awx/blob/devel/awx/main/tasks.py#L1213

This may just be my excuse to use emojis in source code, though.